### PR TITLE
pdfcpu: Add version 0.3.13

### LINKED ISSUE #3257: [Request] Add pdfcpu
### Criteria

For a package to be acceptable in the Main bucket, it should be:

- [X] a non-GUI tool
- [X] a reasonably well-known and widely used developer tool
- [X] the latest stable version of the program
- [X] the full version i.e. not a trial version
- [X] a fairly standard install (e.g. uses a version-specific download URL, no elaborate pre/post install scripts)

### Information

**Name:** pdfcpu

**Description:**
A comprehensive PDF processing library written in Go.
Aims to support at least the standard range of PDF processing features.

**Homepage:**
https://pdfcpu.io
https://github.com/pdfcpu/pdfcpu

**Download link(s):** https://github.com/pdfcpu/pdfcpu/releases

**Some indication of popularity/repute:** 2930+ GitHub stars

### PR DESCRIPTION
**Pull request by febkor**
Closes #3257

- [X] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/pdfcpu.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+    "version": "0.3.13",
+    "description": "A PDF processor written in Go.",
```
Comment (rashil2000) [inline bucket/pdfcpu.json:3]:
```suggestion
    "description": "A PDF processor written in Go, supporting encryption.",
```

File: bucket/pdfcpu.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    "homepage": "https://github.com/pdfcpu/pdfcpu",
```
Comment (rashil2000) [inline bucket/pdfcpu.json:4]:
```suggestion
    "homepage": "https://pdfcpu.io",
```

File: bucket/pdfcpu.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+            "extract_dir": "pdfcpu_0.3.13_Windows_x86_64"
+        },
+        "32bit": {
+            "url": "https://github.com/pdfcpu/pdfcpu/releases/download/v0.3.13/pdfcpu_0.3.13_Windows_i386.zip",
+            "hash": "edecea5d9be7de96b6698957be22553a04a09e8b279c08f9b4c205b93738affa",
+            "extract_dir": "pdfcpu_0.3.13_Windows_i386"
+        }
+    },
+    "bin": "pdfcpu.exe",
+    "checkver": "github",
```
Comment (rashil2000) [inline bucket/pdfcpu.json:19]:
```suggestion
    "checkver": {
        "github": "https://github.com/pdfcpu/pdfcpu"
    },
```

**Review by rashil2000 — CHANGES_REQUESTED** [review]

**Comment by rashil2000** [comment]
In autoupdate, hash logic is same for both architectures, so it can be moved out of architecture block

**Review by tech189 — APPROVED** [review]

**Pull request merged**